docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,112 @@
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+### Advisors
+
+- **Run individual advisor checks**
+
+    In addition to running all available advisors at once, you now have the option to run each advisor check individually. 
+
+    This gives you more granular control over the checks that run on your connected databases. Running checks individually also means that you get the results for relevant advisors faster, and that you can focus on resolving failed checks one at at a time. For more information, see [Working with Advisor checks](https://docs.percona.com/percona-monitoring-and-management/how-to/advisors.html).
+
+- **Advisor checks enabled by default**
+
+    Starting with the previous release and continuing with this one, we have added major improvements to the Advisors Checks functionality in terms of performance, assessment coverage, and user experience. 
+
+    As a mature and generally useful feature, this option is now enabled by default, for  easier access to automatic checks and better insight into database health and performance.
+
+    !!! note alert alert-primary "Note"
+        Upgrading to PMM will automatically enable this feature for existing PMM instances. You can disable it at any time from your PMM Advanced Settings.
+
+- **Enhanced Advisor checks**
+
+    PMM 2.28 includes a new major version of Advisors that features some important enhancements. The most significant changes are:
+
+    * Support for multiple queries
+    * Support for Victoria Metrics as a data source
+
+    In a nutshell, these are changes will allow experts to create more intelligent advisor checks so that we can continue delivering more value to your connected PMM instances.
+
+    The file format in which Advisors checks are written has been updated in order to support the new functionality provided by the Advisors service in Percona Platform. 
+
+    This is a breaking change so it is highly recommended to upgrade your PMM to benefit from these enhancements. For more information see [Develop Advisors](https://docs.percona.com/percona-monitoring-and-management/details/develop-checks.html). 
+
+### Ubuntu 22.04 LTS support
+
+
+### Components upgrade
+  - **VictoriaMetrics**: VictoriaMetrics has been upgraded to [1.76.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.76.1).
+  - **Node exporter**: Node Exporter has now been updated to `1.1.X`.
```

Comment:
@rnovikovP do we need this ? new node exporter suppose to work 1:1 with old one. Maybe there are new features, but we probably first need to find out and adjust dashboards.

Reply:
@rnovikovP Please confirm if we need to skip this or include this...

Reply:
If we did an Update of the exporter - it's a must to be mentioned.

Reply:
```suggestion
  - **Node exporter**: Node Exporter has now been updated to `1.3.1`. Please be aware that if you customized disabled collectors the list could change, please check available collectors in [documentation](https://github.com/percona/node_exporter/blob/main/README.md#collectors).
```

@rasika-chivate we decided to mentioned it and point to the list of new collectors that are available for ppl. We probably need something like: